src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionsBase.cs

```diff
@@ -3361,7 +3361,7 @@ private bool HasUnboxingConversion(TypeSymbol source, TypeSymbol destination, re
             Debug.Assert((object)source != null);
             Debug.Assert((object)destination != null);
 
-            if (destination.IsPointerType())
+            if (destination.IsPointerOrFunctionPointer())
```

Comment:
>if (destination.IsPointerOrFunctionPointer()) [](start = 12, length = 45)

It doesn't look like this change is related to dynamic, but all tree added tests look dynamic specific.

Reply:
Whoops, sorry. I had dynamic commits scattered in a few places and tried to rebase to consolidate, but must have accidentally included this change in the group. There is no specific test covering this, as this is just an easy-out optimization and this function would return false regardless of whether it does so quickly or slowly.

---
In reply to: [407692633](https://github.com/dotnet/roslyn/pull/43248#discussion_r407692633) [](ancestors = 407692633)